packages/fury-adapter-oas3-parser/test/integration/fixtures/petstore.sourcemap.json

```diff
@@ -251,31 +251,6 @@
                       "content": {
                         "key": {
                           "element": "string",
-                          "attributes": {
```

Comment:
This is a regression, source maps for the name are being removed.

Above we have the line:

```js
parameter.set('name', encodeQueryName(parameter.get('name').toValue()));
```

This should become something like the following so we can retain the sourcemaps in the "name" element:

```js
const name = parameter.get('name');
name.content = encodeQueryName(name.toValue());
```